### PR TITLE
Add "tokens" string array to _experience.decisioning.propositionAction object

### PR DESCRIPTION
**Pull request by krishnanbalasubramanyam**
Please link to the issue #1826 

The tokens are required for AJO Inbound decision item tracking.

### CODE REVIEW

File: extensions/adobe/experience/decisioning/proposition-action.schema.json [hunk 1]
```diff
@@ -26,6 +26,14 @@
           "title": "label",
           "description": "The customer generated display name of the element with which the end-user interacted.",
           "type": "string"
+        },
+        "xdm:tokens": {
+          "title": "tokens",
+          "type": "array",
+          "description": "An array of decision item tokens for tracking",
```
Comment (ewiener) [inline extensions/adobe/experience/decisioning/proposition-action.schema.json:33]: Let’s use a more solution-agnostic description consistent with the description for id and labels, e.g. “Application-generated metadata associated with the element with which the end-user interacted.”

Comment (krishnanbalasubramanyam) [reply extensions/adobe/experience/decisioning/proposition-action.schema.json]: Done.

File: extensions/adobe/experience/decisioning/proposition-action.schema.json [hunk 2]
```diff
@@ -26,6 +26,14 @@
           "title": "label",
           "description": "The customer generated display name of the element with which the end-user interacted.",
           "type": "string"
+        },
+        "xdm:tokens": {
```
Comment (anandphatak) [inline extensions/adobe/experience/decisioning/proposition-action.schema.json:30]: Can this field be named a bit more semantically meaningful?

Comment (krishnanbalasubramanyam) [reply extensions/adobe/experience/decisioning/proposition-action.schema.json:30]: Tokens refer to the proposition-interaction tokens. The field "tokens" is enclosed with proposition interaction, just like "label" and "id" that is already present.

**Review by anandphatak — APPROVED** [review]

**Pull request merged**